### PR TITLE
fix arrow position

### LINKED ISSUE #476: Fix position of arrows on arrow links
At least in Safari the arrows are a bit off:

Bildschirmfoto 2019-05-21 um 09 13 40

### PR DESCRIPTION
**Pull request by ghislaineguerin**
closes #476 

Please review

### CODE REVIEW

**Review by pichfl — APPROVED** [review]
I prefer using `transform: translate(0, -50%)` for most cases, but since we control the size, `margin` works fine 👍

**Review by marcoow — COMMENTED** [review]
I can't see a difference between this and `new-simplabs` (in Safari)?

File: src/ui/components/ArrowLink/stylesheet.css [hunk 1]
```diff
@@ -19,7 +19,8 @@
 
 .svg {
   position: absolute;
-  top: 1rem;
+  top: 50%;
+  margin-top: -2.5rem;
```
Comment (pichfl) [inline src/ui/components/ArrowLink/stylesheet.css]: This should move the arrow to the center of the x-height in all browsers:

```suggestion
  margin-top: -2.75rem;
```

**Review by pichfl — CHANGES_REQUESTED** [review]

**Review by marcoow — APPROVED** [review]
👍

**Review by pichfl — APPROVED** [review]

**Pull request merged**